src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,31 @@
 package core.basesyntax;
 
+import core.basesyntax.figures.Figure;
+import core.basesyntax.figures.figurehandlers.FigureSupplier;
 /**
  * Feel free to remove this class and create your own.
  */
+
 public class HelloWorld {
 
+    private static final FigureSupplier figureSupplier = new FigureSupplier();
```

Comment:
there is no need to declare this static variable

since you're using `FigureSupplier` in the `main` method - you can create its instance directly there

making such variables makes sense when you use them in methods, that are often invoked (which is not the case of `main` method) - this way you avoid creation of new instances each time the method is invoked

see [here](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level)